packages/aws-cdk-lib/aws-s3-assets/test/asset.test.ts

```diff
@@ -143,11 +143,18 @@ test('"readers" or "grantRead" can be used to grant read permissions on the asse
   });
 });
 
+test('fails if path is empty', () => {
+  const stack = new cdk.Stack();
+  expect(() => new Asset(stack, 'MyDirectory', {
+    path: '',
+  })).toThrow(/Asset path cannot be empty/);
+});
+
 test('fails if directory not found', () => {
   const stack = new cdk.Stack();
   expect(() => new Asset(stack, 'MyDirectory', {
     path: '/path/not/found/' + Math.random() * 999999,
-  })).toThrow();
+  })).toThrow(/Cannot find asset/);
```

Comment:
this just throws inherently, correct? Possible to catch this in `asset.ts` and add the path: `Cannot find asset at path: <path>` ?

Reply:
similar to https://github.com/aws/aws-cdk/blob/e63c7770ea0a7ec12353973cdf64015f9a4121f1/packages/aws-cdk-lib/core/lib/asset-staging.ts#L174-L176

Reply:
I just wanted to refine the existing test to help differentiate between the two errors, since I was adding a new one.

Reply:
I see it now 👍🏼